### ISSUE TITLE
Propose better set of colors

### ISSUE THREAD

**Issue by bacloud14** [bacloud14/Classified-ads-48#9]
Colors are important for accessibility and for a better user experience. This issue is about proposing a set of colors or going through the CSS part of it. Both regular and dark mode should be considered.

**Comment by bacloud14** [comment]
Also making a new set of colors acting on Bootstrap (if possible) would give a unique taste.

**Comment by Paawan9700** [comment]
hey, can you assign this issue to me, i have much experience in UI and CSS and i will put my efforts to resolve it.

**Comment by bacloud14** [comment]
Hello @Paawan9700 ,
Let's give it a try then ! 😄

**Comment by mitalsapkale01** [comment]
@bacloud14 
Can more than one person works on this issue?, I'm interested for this.

**Comment by bacloud14** [comment]
I prefer not to, I dealt especially with the case of style that involves CSS files particularly and it's always "a step on others toes" situation. 
Let's wait for @Paawan9700 for now. 
@Paawan9700 is it going well? 😁
